defold-spine/editor/src/spineext.clj

```diff
@@ -363,7 +377,7 @@
                                                           (:view render-args)
                                                           (:projection render-args)
                                                           (:texture render-args)))]
-
+    (gl/set-blend-mode gl blend-mode)
```

Comment:
We might want to change the `.glBlendFunc` call below that resets the blend state to `(gl/set-blend-mode :blend-mode-alpha)` so that there is symmetry. I didn't clock it, and was about to ask if we shouldn't reset the blend mode when we're done. 😅